jdaviz/configs/default/plugins/export/tests/test_export.py

```diff
@@ -348,3 +348,45 @@ def test_ap_phot_plot_export(self, imviz_helper):
 
         # just check that it doesn't crash, since we can't download
         export_plugin.export()
+
+    def test_figure_export(self, imviz_helper):
+
+        data = NDData(np.ones((500, 500)) * u.nJy)
+
+        imviz_helper.load_data(data)
+
+        export_plugin = imviz_helper.plugins['Export']._obj
+
+        export_plugin.export(filename=None)
+
+        # attempt to save a figure back to back
+        try:
+            export_plugin.export(filename='img.png')
+        except ValueError as e:
+            assert str(e) == "previous png export is still in progress. Wait to complete before making another call to save_figure"  # noqa: E501
+
+    def test_filepath_convention(self, imviz_helper):
+
+        data = NDData(np.ones((500, 500)) * u.nJy)
+
+        imviz_helper.load_data(data)
+
+        export_plugin = imviz_helper.plugins['Export']._obj
+
+        export_plugin.filename_value = '/img.png'
+        assert os.path.abspath(export_plugin.default_filepath) == os.path.abspath(export_plugin.filename_value)  # noqa: E501
+
+        export_plugin.filename_value = '~/img.png'
+        expected_path = os.path.normpath(os.path.expanduser('~/img.png'))
+
+        assert export_plugin.default_filepath == expected_path
+
+        export_plugin.filename_value = '../img.png'
+        expected_path = os.path.abspath('../img.png')
+
+        assert export_plugin.default_filepath == expected_path
+
+        export_plugin.filename_value = './img.png'
+        expected_path = os.path.abspath('./img.png')
```

Comment:
You might want to create these filenames with OS independent methods for building paths like `Path('.') / 'image.png'`.

Reply:
I'm all for these changes, thank you for comment (and the one above), both will be implemented shortly!